lib/pattern-importer.js

```diff
@@ -0,0 +1,155 @@
+var through = require('through2'),
+  utility = require('./utility'),
+  merge = require('lodash.merge'),
+  gutil = require('gulp-util'),
+  path = require('path'),
+  fs = require('fs'),
+  PluginError = gutil.PluginError,
+  PLUGIN_NAME = 'pattern-importer';
+
+/*
+ * Imports html patterns that are following the standards at github.com/pattern-library
+ *
+ * @param {Object} options
+*/
+exports.patternImporter = function (options) {
```

Comment:
@e2tha-e Can you please add your unit test to `test/main.js` and issue a PR? Might as well capture it while it's hot!